package/src/external/reanimated/moduleWrapper.ts

```diff
@@ -0,0 +1,45 @@
+import { useMemo } from "react";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let Reanimated: any;
+
+try {
+  Reanimated = require("react-native-reanimated");
+} catch (e) {
+  // Ignore
+}
+
+export const HAS_REANIMATED = Reanimated !== undefined;
+
+function throwOnMissingReanimated() {
+  throw new Error(
+    "Reanimated was not found, make sure react-native-reanimated package is installed if you want to use react-naitve-skia's integration layer API."
+  );
+}
+
+function isSharedValuePolyfill(value: any) {
```

Comment:
This function is ok. It could be rewritten to take unknown and an input (and use a type assertion function to have value typed as Record<string, unknown>) and it should itself return a type assertion. Definitely not a big deal just mentioning it.

Reply:
We actually added it to reanimated recently so I'll be using the version imported from reanimated instead. It makes better sense as reanimated knows better how shared values can be distinguished